ts/Extensions/Annotations/Popup.ts

```diff
@@ -795,162 +875,381 @@ H.Popup.prototype = {
+                buttonParentDiv as HTMLDOMElement,
+                callback
             );
 
             // EDIT tab
-            this.addColsContainer(tabsContainers[1] as any);
+            this.addColsContainer(tabsContainers[1] as HTMLDOMElement);
             indicators.addIndicatorList.call(
                 this,
                 chart,
-                tabsContainers[1] as any,
+                tabsContainers[1] as HTMLDOMElement,
                 'edit'
             );
 
             buttonParentDiv = tabsContainers[1]
                 .querySelectorAll('.' + PREFIX + 'popup-rhs-col')[0];
 
             this.addButton(
-                buttonParentDiv as any,
+                buttonParentDiv as HTMLDOMElement,
                 lang.saveButton || 'save',
                 'edit',
-                callback,
-                buttonParentDiv as any
+                buttonParentDiv as HTMLDOMElement,
+                callback
             );
             this.addButton(
-                buttonParentDiv as any,
+                buttonParentDiv as HTMLDOMElement,
                 lang.removeButton || 'remove',
                 'remove',
-                callback,
-                buttonParentDiv as any
+                buttonParentDiv as HTMLDOMElement,
+                callback
             );
         },
+
+        /**
+         * Filter object of series which are not indicators.
+         * If the filter string exists, check against it.
+         *
+         * @private
+         *
+         * @param {Highcharts.FilteredSeries} series
+         *        All series are available in the plotOptions.
+         *
+         * @param {string|undefined} filter
+         *        Applied filter string from the input.
+         *        For the first iteration, it's an empty string.
+         *
+         * @return {void}
+         */
+        filterSeries: function (
+            this: Highcharts.Popup,
+            series: Series,
+            filter?: string
+        ): [Highcharts.FilteredSeries] {
+            const popup = this,
+                indicators = popup.indicators;
+            let filteredSeriesArray: [Highcharts.FilteredSeries] = [] as any,
+                filteredSeries: Highcharts.FilteredSeries;
+
+
+            objectEach(series, function (
+                series: Series,
+                value: string
+            ): void {
+                const seriesOptions = series.options;
+                // Allow only indicators.
+                if (
+                    (series as any).params || seriesOptions &&
+                    (seriesOptions as any).params
+                ) {
+                    const { indicatorFullName, indicatorType } =
+                        indicators.getNameType(series, value);
+
+                    if (filter) {
+                        const slicedValue = indicatorFullName.slice(0, filter.length);
+                        if (slicedValue.toLocaleLowerCase() === filter.toLocaleLowerCase()) {
```

Comment:
In other words, typing `stochastic` will find only Stochastic technical indicator, and Slow Stochastic will be missing? Use safe regexp instead